src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -1062,4 +1087,24 @@ public void onBuildingUpgradeComplete(AbstractBuilding building, int level)
     {
         return ServerUtils.getPlayersFromUUID(this.world, this.getPermissions().getMessagePlayers());
     }
+
+    // todo: document!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)